### PR TITLE
Use of our library to read PDF files

### PR DESCRIPTION
**Pull request by tevincent**
Design of the handle is not final.

Depends on #1134

### CODE REVIEW

**Comment by tevincent** [comment]
Uploading new_handle_pdf.mp4…

File: app/src/main/java/com/infomaniak/drive/utils/PreviewPDFUtils.kt [hunk 1]
```diff
@@ -40,7 +40,7 @@ object PreviewPDFUtils {
         file: File,
         userDrive: UserDrive,
         onProgress: (progress: Int) -> Unit
-    ): ApiResponse<PdfCore> {
+    ): ApiResponse<java.io.File> {
```
Comment (JorisBodin) [inline app/src/main/java/com/infomaniak/drive/utils/PreviewPDFUtils.kt:43]: Why don't you use `IOFile` there too?

**Review by JorisBodin — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewPDFFragment.kt [hunk 2]
```diff
@@ -27,32 +27,29 @@ import androidx.core.view.isGone
...
 
 class PreviewPDFFragment : PreviewFragment() {
 
     private var binding: FragmentPreviewPdfBinding by safeBinding()
 
-    private var previewPDFAdapter: PreviewPDFAdapter? = null
     private val previewPDFViewModel by viewModels<PreviewPDFViewModel>()
 
-    private var pdfCore: PdfCore? = null
+    private var pdfFile: java.io.File? = null
```
Comment (JorisBodin) [inline app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewPDFFragment.kt:52]: IOFile

**Review by JorisBodin — CHANGES_REQUESTED** [review]

**Comment by tevincent** [comment]
https://github.com/Infomaniak/android-kDrive/assets/149579879/7af1105e-6a27-4059-9f56-d167729ec875

**Review by JorisBodin — APPROVED** [review]

**Pull request merged**